app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsScreen.kt

```diff
@@ -69,6 +78,38 @@ fun ParkingDetailsScreen(
   val scrollState = rememberScrollState()
   val context = LocalContext.current
   val toastMessage = stringResource(R.string.sign_in_to_add_favorites)
+  // === States for the images ===
+  var newParkingImageLocalPath by remember { mutableStateOf("") }
+  val showDialog = remember { mutableStateOf(false) }
+  val imagesUrls by parkingViewModel.selectedParkingImagesUrls.collectAsState()
+  // === === === === === === ===
+
+  LaunchedEffect(Unit, selectedParking) {
+    // On first load of the screen, request the images
+    // This will update the imagesUrls state and trigger a recomposition
```

Comment:
Why do you have a launched effect of unit and selected parking? Isn't just selected parking enough?

Reply:
I'll add a comment to make it clearer !